llarp/service/context.cpp

```diff
@@ -92,6 +92,8 @@ namespace llarp
           expired.emplace_back(rc.pubkey);
         return true;
       });
+      // TODO: we need to stop looking up service nodes that are gone forever
+      // how do?
```

Comment:
well you need to know they're gone forever and we can never know that. Need to adjust your where clause there. I imagine some type of timestamp threshold comes into play here

Reply:
correct, this is a problem of propagating consensous state to lokinet clients